lib/DatabaseLayer.py

```diff
@@ -203,6 +216,8 @@ def userExists(user):
 
 def isSingleMaster(user):
   return True if len(list(colUSERS.find({"username": {"$ne": user}, "master": True}))) == 0 else False
```

Comment:
`return len(list(colUSERS.find({"username": {"$ne": user}, "master": True}))) == 0`

Reply:
It is likely not necessary to construct an entire list from the `find`, you could just use `next` builtin to verify that at least 1 element can be constructed, in which case your condition would fail...